README.md

```diff
@@ -1,12 +1,14 @@
 # Content Sources
 
 ## What is it?
-
 Content Sources is an application for storing information about external content (currently YUM repositories) in a central location.
 
 
 ## Developing
 
+### Requirements:
+1. podman installed or docker docker installed (and running)
```

Comment:
says "docker docker" instead of "docker"

Reply:
roger roger!